docs/More Advice/3.Declaring a Minor.md

```diff
@@ -0,0 +1,31 @@
+# Declaring a Minor
+
+## What is a Minor?
+A minor represents a supplementary field of study beyond your designated major. It is not an additional degree conferred at graduation but an extra qualification appended to your degree upon completion. Declaring a minor is not required for graduation, however, students who have a particular interest in a field or feel that a minor would be beneficial to their degree typically declare a minor. 
+
+## Should I Declare an Undergraduate Minor? 
+Taking on an additional program of study like a minor can be a lot of extra work, minors range from 15 to 21 (maximum) credit hours of additional work on top of your major. Classes for a minor may or may not count towards classes for your major, be it core courses or free electives, so keep this in mind when making your decision. Minors can also be used in a James Scholar plan or honors contract, many students also declare a minor for this purpose. Most importantly, make sure that your minor is also in a field that you enjoy, students declare minors for a vareity of reasons but the main reason people drop or cancel their minor is because they do not enjoy the field.
+
+## Taking Classes for my Undergraduate Minor
+Taking classes for your minor can be complicated at times, especially because most departments do not offer priority registration for students pursuing a minor. While courses are typically limited to majors and minors of the host department, their registration timelines differ. Departments will (almost) always prioritize students who are pursuing a major in their department, so taking popular classes in the department can prove to be difficult. The best advice one can give for registering the classes you want while pursuing minor is to make a plan. Familiarize yourself with class registration dates and set notifications using the course explorer.
```

Comment:
this writing can be simplified, the first two sentences seem like duplicates of each other